pkg/observability/package.json

```diff
@@ -18,11 +19,6 @@
   "engines": {
     "node": ">=12"
   },
-  "devDependencies": {
-    "@vue/cli-plugin-babel": "4.5.18",
-    "@vue/cli-service": "4.5.18",
-    "@vue/cli-plugin-typescript": "4.5.18"
-  },
```

Comment:
works fine without them 🤔

Reply:
@owanturist It doesn't...

Regarding extensions the definitive test is always doing a developer load, which is the closest to a PROD environment you'll get. To achieve that, you need to do a build locally https://extensions.rancher.io/extensions/extensions-getting-started#building-the-extension (which fails if you don't have these dev dependencies), then you do the actual developer load https://extensions.rancher.io/extensions/extensions-getting-started#test-built-extension-by-doing-a-developer-load.

I recommend everyone to do this after you've finished your work locally, to make sure it all works on a desired system. Local env is slightly different of a "PROD" env because the way it loads extensions and how it handles it's dependencies.

Reply:
There's a pipeline where we can include this build, yes. It's tied to the CI/CD of the dashboard repo https://github.com/rancher/dashboard/blob/master/.github/workflows/check-plugins.yaml, more precisely https://github.com/rancher/dashboard/blob/master/shell/scripts/test-plugins-build.sh (towards the end of the file).

I do recommend that you branch **before** you merge this commit (something along the lines of `release-pre-2.9` or `release-2.8`) so that it tracks the correct shell version + annotations for everything pre-2.9 and leave the `main` branch with these changes that you want to get in. 

You can later port them to that branch (except the shell + annotation changes) so that both branches are up to par.

Reply:
did I get it right that you recommend to rename the branch (or at least change the target from name to `release-pre-2.9` and keep those changes away from `main`?

Reply:
ah, now when I read it again I got that you recommend to keep track of the legacy version

do you think a git tag would be good enough to keep track of it? a branch can be deleted easier and it is not useful as long as it does not include any additional changes?